tests/fixtures/integration_fixtures.py

```diff
@@ -149,17 +149,21 @@ def postgres_inserts(postgres_integration_db):
 # ======================= mongodb  ==========================
 
 
-@pytest.fixture(scope="session")
-def integration_mongodb_config() -> ConnectionConfig:
-    return ConnectionConfig(
+@pytest.fixture(scope="function")
+def integration_mongodb_config(db) -> ConnectionConfig:
+    connection_config = ConnectionConfig(
         key="mongo_example",
         connection_type=ConnectionType.mongodb,
         access=AccessLevel.write,
         secrets=integration_secrets["mongo_example"],
+        name="mongo_example",
     )
+    connection_config.save(db)
+    yield connection_config
+    connection_config.delete(db)
```

Comment:
This connectionconfig wasn't being saved to the db previously, which was fine, but now I'd like to be able to persist it to test disabling it mid-privacy request, for example, and I'm changing the fixture scope so updates to disabled are reset for another test.